sympy/geometry/tests/test_line.py

```diff
@@ -53,6 +53,14 @@ def test_angle_between():
                                 Line3D(Point3D(0, 0, 0), Point3D(5, 0, 0))), acos(sqrt(3) / 3)
 
 
+def test_closing_angle():
```

Comment:
I wonder why this is showing up here when it was add [already](https://github.com/sympy/sympy/pull/13002/files).